### PR TITLE
[Cow] Make cow community spells trino visible

### PR DESCRIPTION
**Pull request by dsalv**
Brief comments on the purpose of your changes:

**For Dune Engine V2**

I've checked that:

### General checks:
* [x] I tested the query on dune.com after compiling the model with dbt compile (compiled queries are written to the target directory)
* [ ] I used "refs" to reference other models in this repo and "sources" to reference raw or decoded tables 
* [ ] if adding a new model, I added a test
* [ ] the filename is unique and ends with .sql
* [ ] each sql file is a select statement and has only one view, table or function defined  
* [ ] column names are `lowercase_snake_cased`
* [ ] if adding a new model, I edited the dbt project YAML file with new directory path for both models and seeds (if applicable)
* [ ] if wanting to expose a model in the UI (Dune data explorer), I added a post-hook in the JINJA config to add metadata (blockchains, sector/project, name and contributor Dune usernames)

### Pricing checks:
* [ ] `coin_id` represents the ID of the coin on coinpaprika.com
* [ ] all the coins are active on coinpaprika.com (please remove inactive ones)

### Join logic:
* [ ] if joining to base table (i.e. ethereum transactions or traces), I looked to make it an inner join if possible

### Incremental logic:
* [ ] I used is_incremental & not is_incremental jinja block filters on both base tables and decoded tables
  * [ ] where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to base table (i.e. ethereum transactions or traces), I applied join condition where block_time >= date_trunc("day", now() - interval '1 week')
* [ ] if joining to prices view, I applied join condition where minute >= date_trunc("day", now() - interval '1 week')

### CODE REVIEW

**Review by bh2smith — APPROVED** [review]
YES!

**Review by jeff-dude — APPROVED** [review]
fine with me, we'll let @belen-pruvost be the final call :)

File: models/cow_protocol/ethereum/cow_protocol_ethereum_app_data.sql [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {{ config(alias='app_data',
-        post_hook='{{ expose_spells_hide_trino(\'["ethereum"]\',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
```
Comment (belen-pruvost) [inline models/cow_protocol/ethereum/cow_protocol_ethereum_app_data.sql:2]: The problem is that if we expose this as is on DuneSQL, it won't work.
 `select * from cow_protocol_ethereum.app_data limit 1;` throws `Unknown type: numeric at line 1, position 15` - could you fix the underlying issue before exposing it?

File: models/cow_protocol/ethereum/cow_protocol_ethereum_order_rewards.sql [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {{ config(alias='order_rewards',
-        post_hook='{{ expose_spells_hide_trino(\'["ethereum"]\',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
```
Comment (belen-pruvost) [inline models/cow_protocol/ethereum/cow_protocol_ethereum_order_rewards.sql:2]: This one is good to go! 🚀

**Review by belen-pruvost — CHANGES_REQUESTED** [review]

File: models/cow_protocol/ethereum/cow_protocol_ethereum_app_data.sql (hunk 1 above)
Comment (dsalv) [reply models/cow_protocol/ethereum/cow_protocol_ethereum_app_data.sql:2]: isn't numeric supported? The issue before was that data wasn't queryable by dunesql, the raw data is queryable now

Comment (dsalv) [reply models/cow_protocol/ethereum/cow_protocol_ethereum_app_data.sql:2]: @bh2smith Would bigint be enough for this parameter?

`BIGINT[#](https://trino.io/docs/current/language/types.html#bigint)

A 64-bit signed two’s complement integer with a minimum value of -2^63 and a maximum value of 2^63 - 1.`

Comment (bh2smith) [reply models/cow_protocol/ethereum/cow_protocol_ethereum_app_data.sql:2]: I believe this value can be just a regular integer. It's bonded between 0 and 10^5.

Comment (dsalv) [reply models/cow_protocol/ethereum/cow_protocol_ethereum_app_data.sql:2]: @bh2smith it's[ integer now ](https://github.com/duneanalytics/spellbook/pull/2799/commits/5697a0338e18024636f78a3466e63f6b52bae601)

**Review by belen-pruvost — APPROVED** [review]
It's working nicely! See [here](https://dev.dune.com/queries/60834?d=11) - thanks for the fixes!

**Review by jeff-dude — APPROVED** [review]
thanks team!

**Pull request merged**